Doc/user_manual/six.tex

```diff
@@ -856,7 +856,7 @@ \section{Special Elements}
 
 One advantage of SixTrack, that has been adopted from RACETRACK, is
 that it easily allows to define elements for a specific purpose. The
-special elements implemented till now are found in this section. All Special Elements should be written in the fort.3 file.
+special elements implemented till now are found in this section.\n All Special Elements should be written in the fort.3 file.
```

Comment:
Hi,

I didn't mean a literal "\n", I meant just a newline (an "enter"), i.e. like this:
```
special elements implemented till now are found in this section.
All Special Elements should be written in the fort.3 file.
```

Can you fix that, then commit and push six.tex and the pdf again?